Debugger Display Not Functioning in Release Build

The Debugger Display is not functioning in Release builds.

**Steps to reproduce**
-1- start debugging (in Debug mode - of course)
-2- hover over the Validator object, after hitting a breakpoint after it in the code
-3- behold the following message : _error CS0103: The name 'ShowCounts' does not exist in the current context_

--- a/src/BusinessValidation/BusinessValidation/Validator.cs
+++ b/src/BusinessValidation/BusinessValidation/Validator.cs
@@ -315,7 +315,6 @@ namespace BusinessValidation
                 throw new ValidationFailureException(ValidationFailures);
         }
 
-#if DEBUG
         private string ShowCounts
         {
             get
@@ -323,7 +322,6 @@ namespace BusinessValidation
                 return $"Failure Count ➡️ {Results.FailCount}, Message Count ➡️ {Results.MessageCount}";
             }
         }
-#endif
 
         private static string ExtractPath<T,TM>(Expression<Func<T, TM>> expression, PropertyDepth propertyDepth) => propertyDepth switch
         {

